cwf/gateway/configs/sessiond.yml

```
@@ -64,3 +64,6 @@ poll_stats_interval: 5
 
 # set to true to enable pull model for stats(polling pipelined from sessiond)
 enable_pull_stats: false
+
+# set to the maximum amount of UEs that can be assigned a particular shard id
```

```suggestion
maximum number of UEs that can be assigned a particular shard id
```
for all the config files